lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -19,13 +17,15 @@ class SmsSender
       # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/BlockLength
       # @return [Response]
       def send(message:, to:)
-        return handle_config_failure if client_configs.empty?
+        return handle_config_failure if Telephony.config.pinpoint.sms_configs.empty?
 
         response = nil
```

Comment:
The above line covers the case if our config is empty, but I think we need to handle having configs but STS failing both times? I don't love it, but we could set this line to a default response that's similar to the one in `handle_config_failure`, and it would get returned at the bottom if the loop below doesn't set a `response`.

Reply:
I think I got that covered in 81f3063